pkg/leaderelection/types.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
```

Comment:
Can we move this file to pkg/types directory? That directory contains all the types

Reply:
done